routes/auth-routes.js

```diff
@@ -1,140 +0,0 @@
-// The routes file will be defined in the routes/auth.routes.js, and we will set the necessary packages and code to signup in the application:
-
-const { Router } = require("express");
-const router = new Router();
-const passport = require("passport");
-const User = require("../models/user");
-const bcrypt = require("bcrypt");
-const bcryptSalt = 10;
-const { isLoggedIn, isLoggedOut } = require("../middleware/route-guard");
-const filteredDomains = require("../public/javascripts/filteredDomains");
-const nodemailer = require("nodemailer");
-const nodemailerSetup = require('../public/javascripts/nodemailer');
```

Comment:
I'm not entirely sure why your importing from your `public/javascripts`, I assume it's some sort of workaround. Generally speaking, your `public/javascript` is your frontend JS code and you should not be importing FE code into your backend

Reply:
Got it. For general 'utility' or 'helper' files/modules, where's it best to place them?